package.json

```diff
@@ -29,6 +29,7 @@
     "google-protobuf": "^3.9.1",
     "npm-run-all": "^4.1.5",
     "ramda": "^0.26.1",
+    "vsce": "^1.78.0",
```

Comment:
No need to add `vsce` as a dependency. Even better you can leave out dependency updates from this PR because they are not necessary.

Reply:
I can't figure out how to revert that file :(

Reply:
i guess i can remove vsce